memod doesn't work with slack

I created an application in Slack, gave the bot all permissions available, enabled the socket, and enabled event subscription. Memod is crashing, every time I type a message in chat, it doesn't matter which message. There are logs:

time="2023-06-26T09:06:22Z" level=info msg="Can talk to Grafana version 9.3.2 - its database is ok"
time="2023-06-26T09:06:22Z" level=info msg="Memo starting"
time="2023-06-26T09:06:22Z" level=info msg="slack enabled"
time="2023-06-26T09:06:22Z" level=info msg="Connecting to slack"
time="2023-06-26T09:06:23Z" level=info msg="Socket connected"
time="2023-06-26T09:06:23Z" level=info msg="Received hello from slack, hi!"
time="2023-06-26T09:06:38Z" level=trace msg="Received message `\"test\"`, not for us. ignoring"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x6e9431]

goroutine 33 [running]:
github.com/grafana/memo.(*Memo).BuildTags(...)
	/opt/memo/memo.go:36
github.com/grafana/memo/service/slack.(*SlackService).handleMessage(0xc0002344d0, 0xc0001668c0)
	/opt/memo/service/slack/slack.go:98 +0x2d1
github.com/grafana/memo/service/slack.New.func1()
	/opt/memo/service/slack/slack.go:162 +0x413
created by github.com/grafana/memo/service/slack.New
	/opt/memo/service/slack/slack.go:133 +0x3e5

Tried to use 1.20 Go version and nothing really helps. Is memo capable to work with slack today, or maybe the code is outdated?

--- a/service/slack/slack.go
+++ b/service/slack/slack.go
@@ -89,6 +89,11 @@ func (s *SlackService) handleMessage(msg *slackevents.MessageEvent) error {
 		return err
 	}
 
+	// message is not for us, but not an error, just bail out
+	if memo == nil {
+		return nil
+	}
+
 	tags := []string{
 		"author:" + usr,
 		"chan:" + ch,
@@ -114,6 +119,7 @@ func New(config cfg.Slack, parser parser.Parser, store store.Store) (service.Ser
 		appToken: config.AppToken,
 
 		parser: parser,
+		store:  store,
 
 		chanIdToNameCache: make(map[string]string),
 		userIdToNameCache: make(map[string]string),

